Python/fitnesse/html_util.py

```diff
@@ -1,10 +1,86 @@
 from io import StringIO
 
-from fitnesse.context import SuiteResponder, PageCrawlerImpl, PageData, PathParser, WikiPage, WikiPagePath
+from fitnesse.context import (
+    SuiteResponder,
+    PageCrawlerImpl,
+    PageData,
+    PathParser,
+    WikiPage,
+    WikiPagePath,
+)
+
+
+class HtmlProcessData:
+    page_data: PageData
+    page_crawler: PageCrawlerImpl
+    path_parser: PathParser
 
 
 class HtmlUtil:
+    @staticmethod
+    def _suite_action(
```

Comment:
This name may not be proper to understand what the function is doing

Reply:
Overall, I would say that this refactor does not necessarily improve the readability, it only gets rid of the code duplication